layers/auto-completion/README.org

```diff
@@ -97,7 +97,7 @@ the popup, set the variable =auto-completion-enable-snippets-in-popup= to =t=.
 #+BEGIN_SRC emacs-lisp
   (setq-default dotspacemacs-configuration-layers
                 '((auto-completion :variables
-                                   auto-completion-enable-snippets-in-popup t))
+                                   auto-completion-enable-snippets-in-popup t)))
```

Comment:
@robbyoconnor No, the new one is correct. You need parenthesis around the layer when there are `:variables`.